src/components/Sections/ItemsSection.js

```diff
@@ -161,18 +161,25 @@ class ItemsSection extends Component {
                   width: colSpan * columnWidth
                 };
                 const type = item.fieldType || '';
-                let dataDisplay = Array.isArray(item.data) ?
-                  <SectionTable data={item.data} /> :
-                  <SectionMarkdown
-                    text={String(item.data)}
-                    markdownArtifactsServerAddress={markdownArtifactsServerAddress}
-                  />;
-                if (type === SECTION_ITEM_TYPE.html) {
-                  dataDisplay = <SectionHTML text={item.data} />;
+                let dataDisplay = item.data;
+                if (Array.isArray(item.data)) {
+                  if (item.data.length === 0) {
+                    return null;
+                  }
+                  dataDisplay = (<SectionTable data={item.data} />);
+                } else if (type === SECTION_ITEM_TYPE.markdown) {
```

Comment:
@idanizi-panw isn't that a regression? text fields will no longer be considered markdown

Reply:
fixed